Dockerfile

```diff
@@ -285,9 +285,6 @@ RUN sudo git clone https://github.com/OSGeo/PROJ.git /home/carma/PROJ --branch 6
 # Download a cmake module for PROJ
 RUN cd /usr/share/cmake-3.16/Modules && sudo curl -O https://raw.githubusercontent.com/mloskot/cmake-modules/master/modules/FindPROJ4.cmake
 
-# Install Autoware.Auto Dependencies
-RUN sudo apt-get install --yes ${AUTOWAREAUTO_DEPS}
-
 # Install Novatel OEM7 Driver Wrapper Dependency
 RUN sudo apt-get install -y ros-foxy-gps-msgs
```

Comment:
Is there a reason this one wasn't rolled into the above deps list?

Reply:
> Is there a reason this one wasn't rolled into the above deps list?

I will in a follow up commit because I wanted  to give it a dedicated commit for history so that the reason this is a dependency is preserved. Because I will have to delete the comment when I do, keeping the comment `#` would break the variable.

Unless you think that's overkill, I can move it with this commit if you'd like.